src/fty_alert_actions.cc

```diff
@@ -793,7 +793,8 @@ s_handle_stream_deliver_asset(fty_alert_actions_t *self, fty_proto_t **asset_p,
     const char *operation = fty_proto_operation (asset);
     const char *assetname = fty_proto_name (asset);
 
-    if (streq (operation, "delete")) {
+    if (streq (operation, "delete")
```

Comment:
while you're editing the sources, could you please update "delete" to FTY_PROTO_ASSET_OP_DELETE?